pom.xml

```diff
@@ -463,6 +464,17 @@
                 <groupId>io.confluent</groupId>
                 <artifactId>broker-plugins</artifactId>
                 <version>${kafka.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>io.confluent</groupId>
+                        <artifactId>cc-role-definitions</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>io.confluent</groupId>
+                <artifactId>cc-role-definitions</artifactId>
```

Comment:
do we need cc-role-definitions jar in ksql docker image? 🤔 
we see this was introduced only in 7.8.x and comes in as transitive dependency..

Reply:
We do not need this but it is runtime dependency for broker-plugins. If we will not include, something might break at runtime due to missing implementation.

https://staging-packages.confluent.io/maven/io/confluent/broker-plugins/7.8.0-ce/
https://staging-packages.confluent.io/maven/io/confluent/broker-plugins/7.8.0-ce/#:~:text=ce.module.sha512-,broker%2Dplugins%2D7.8.0%2Dce.pom,-broker%2Dplugins%2D7.8.0